src/styles/utilities/_extends.scss

```diff
@@ -0,0 +1,23 @@
+%size-description-code {
```

Comment:
The name of the placeholder is not descriptive enough. You should use a consistent naming convention that reflects the content or purpose of the style. For example, consider renaming this to '%description-code-style'.